### ISSUE TITLE
Auto-merge vets-website (va-gov/web) packagist updates

### ISSUE THREAD

**Issue by ElijahLynn** [department-of-veterans-affairs/va.gov-cms#910]
AC 
- [ ] The latest tag from vets-website is auto-merged by dependabot on a daily basis, at a time that does not interfere with CMS deployments.

**Comment by ElijahLynn** [comment]
Update: I removed the va-gov/web from the .dependabot/config.yml that we thought was causing issues, https://github.com/department-of-veterans-affairs/va.gov-cms/pull/1634, and now it fails on a simplesaml failure the same way, which is not a git package (I think). So I am not sure the original triage of our issue is valid so don't make that assumption when resolving this. 

Here is most recent log failure:
```
proxy | 2020/05/13 14:13:55 200 https://packages.drupal.org/8/drupal/provider-2020-2%24df1b51c8f06df6f70684a0eec4e6953166fc3344ed201bb3392f61240f190820.json
updater | INFO <job_33011188> Requirements to unlock update_not_possible
updater | INFO <job_33011188> Requirements update strategy bump_versions_if_necessary
updater | INFO <job_33011188> No update possible for simplesamlphp/simplesamlphp 1.17.8
updater | INFO <job_33011188> Finished job processing
```

**Comment by indytechcook** [comment]
The main issue is with the tags not making it to packagist.  If the tags where there then dependable would work correctly.

**Comment by indytechcook** [comment]
Here is a message I sent to the vets-website team regarding the tag in git.

> I've found a problem with how the vets-website git tags are names and integration with packagist (npm for php ).  We use packagist from within the Drupal repo to keep pull versions of the vets-website into our project for tests/builds and CI.  The issue lies with the name of the tag vets-website/v0.1.931.  Packagist expects v0.1.931 or an update to the composer.json file.  I'd like to test this theory by manually adding a tag to the repo but I wanted to check first to see if that would break anything in BRD.  Or if anyone has issues with me updating the vets-website BRD to tag vets-website/v0.1.931 and v0.1.931 in git.  If it's not ok, there is a manually way to interact with the packagist API but this way is easier.

**Comment by indytechcook** [comment]
I added a tag manually to vets-website and it did make it to packagist:  

![image.png](https://images.zenhubusercontent.com/543e76bb7c6ccc6277a54276/d7b4a30b-d054-43ce-bd7f-419f010ddf26)

**Comment by indytechcook** [comment]
This requires a bit more work after some feedback from ops so I"m moving to the next sprint.

**Comment by indytechcook** [comment]
PR to use tags instead of dev-master: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/2136

**Comment by indytechcook** [comment]
The previous work cause some failures on the release job.    New PR to add error handling: https://github.com/department-of-veterans-affairs/devops/pull/7212

**Comment by indytechcook** [comment]
The auto tagging is now working.  There is still an issue with the tag based builds not working with the current vets website build.

**Comment by indytechcook** [comment]
With PR #2322 we are able to keep the git repo while using versions.  This should work with dependabot.  This PR is in review because we are waiting on dependabot to send in a PR.

**Comment by indytechcook** [comment]
Worked!  https://github.com/department-of-veterans-affairs/va.gov-cms/pull/2377

**Comment by ElijahLynn** [comment]
We are still waiting to turn on the auto-merge portion, correct?

**Comment by ElijahLynn** [comment]
I see in .github/dependabot.yml that we don't have this set to auto-merge just yet. Let's do that and then we can close this.

**Comment by ElijahLynn** [comment]
Booohooooo > https://github.com/dependabot/feedback/issues/954#issuecomment-640919061

> Auto-merge will not be supported in GitHub-native Dependabot for the foreseeable future. We know some of you have built great workflows that rely on auto-merge, but right now, we’re concerned about auto-merge being used to quickly propagate a malicious package across the ecosystem. We recommend always verifying your dependencies before merging them.

**Comment by Greendogo** [comment]
@ElijahLynn Did you find another solution to work-around Dependabot not supporting Auto-Merge?

**Comment by ElijahLynn** [comment]
> @ElijahLynn Did you find another solution to work-around Dependabot not supporting Auto-Merge?

Unfortunately we did not, although did look at a few other options like Renovate and Violinist (specifically for PHP/Composer) but didn't end up going that route and instead are still manually merging all the Dependabot PRs.
